_includes/supporter.html

```diff
@@ -0,0 +1,16 @@
+<div id="sponsor" class="page-wrapper sponsor-section">
+  <div class="page-details">
+    <div class="title">Sponsors</div>
+    <div class="description">
+      PyCon India is entirely driven by volunteers.
+      Sponsoring the event helps to sustain and grow
+      the conference as well as the community.
+      Sponsors help in making the conference affordable,
+      and maintaining the inventory for the conference.
```

Comment:
PyCon India is volunteer driven event. Sponsoring the event helps to sustain and grow
the conference and the community. Sponsors help in making the conference affordable,
and maintain the inventory for the conference.

Reply:
Let's create a PR out of this so everyone can comment. cc: @vipulgupta2048

Reply:
Created an issue to track the discussion: https://github.com/pythonindia/inpycon2020/issues/18